examples/simple-repl.rs

```diff
@@ -1,11 +1,12 @@
 use jinko::context::Context;
+use jinko::io_trait::JkStdReader;
 use std::io::{self, Result, Write};
 
 fn main() -> Result<()> {
     let stdin = io::stdin();
     let mut stdout = io::stdout();
 
-    let mut ctx = Context::new();
+    let mut ctx = Context::new(Box::new(JkStdReader {}));
```

Comment:
Here for example, we'll be able to avoid the extra curly brackets

Reply:
Fixed by 282dce49e18f7cfb41903a2abce277c4d0df57df